common/model-views.cpp

```diff
@@ -4690,6 +4874,49 @@ namespace rs2
                 }
             }
 
+
+            if (dev.supports(RS2_CAMERA_INFO_PRODUCT_LINE) && dev.supports(RS2_CAMERA_INFO_FIRMWARE_VERSION))
+            { 
+                auto product_line_str = dev.get_info(RS2_CAMERA_INFO_PRODUCT_LINE);
+                if (RS2_PRODUCT_LINE_L500 == parse_product_line(product_line_str))
+                {
+                    // Make sure the firmware is at least 1.4.1.2 for Trigger Camera Accuracy features
+                    if (false == is_upgradeable(dev.get_info(rs2_camera_info::RS2_CAMERA_INFO_FIRMWARE_VERSION), "1.4.1.2"))
+                    {
+                        if (ImGui::Selectable("Trigger Camera Accuracy Health"))
+                        {
+                            // We cannot open a pop up window here since we are already in a pop up window
+                            // we trigger the pop up and activate it outside the menu pop up
+                            cah_model.show_trigger_camera_accuracy_health_popup = true;
+                        }
+                        if (ImGui::IsItemHovered())
+                            ImGui::SetTooltip("Trigger Camera Accuracy Health process");
+
+
+                        if (ImGui::Selectable("Reset Camera Accuracy Health"))
+                        {
+                            cah_model.show_reset_camera_accuracy_health_popup = true;
+                        }
+                        if (ImGui::IsItemHovered())
+                            ImGui::SetTooltip("Reset Camera Accuracy Health process outputs");
+                    }
+                    else
+                    {
+                        bool selected = false;
+
+                        ImGui::Selectable("Trigger Camera Accuracy Health",&selected, ImGuiSelectableFlags_Disabled);
+                        if (ImGui::IsItemHovered())
+                            ImGui::SetTooltip("Required firmware versions 1.4.1.0 and above");
+
+
+                        ImGui::Selectable("Reset Camera Accuracy Health", &selected, ImGuiSelectableFlags_Disabled);
+                        if (ImGui::IsItemHovered())
+                            ImGui::SetTooltip("Required firmware versions 1.4.1.0 and above");
```

Comment:
1.4.1.2
Should say "Requires" and "version"
Same in line 4909

Reply:
Will be updated